src/pages/Home/components/MainContent.test.js

```diff
@@ -0,0 +1,17 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import MainContent from './MainContent';
+
+describe('MainContent', () => {
+    test('Should create a container for main content', () => {
+        const item = render(<MainContent />)
+        const homeContainer = item.container.querySelector('.homeDiv')
```

Comment:
instead of `item.container.querySelector`, you should try to use: `screen.[query*|get*]`. See: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence for more advices